internal/controller/scheduler/scheduler.go

```diff
@@ -378,33 +380,9 @@ func (w *jobWrapper) Build() (*batchv1.Job, error) {
 		},
 	}
 	agentContainer.Env = append(agentContainer.Env, env...)
-	// system client container(s)
-	checkoutContainer := corev1.Container{
-		Name:            "checkout",
-		Image:           w.cfg.Image,
-		Args:            []string{"bootstrap"},
-		WorkingDir:      "/workspace",
-		VolumeMounts:    volumeMounts,
-		ImagePullPolicy: corev1.PullAlways,
-		Env: []corev1.EnvVar{{
-			Name:  "BUILDKITE_AGENT_EXPERIMENT",
-			Value: "kubernetes-exec",
-		}, {
-			Name:  "BUILDKITE_BOOTSTRAP_PHASES",
-			Value: "checkout,command",
-		}, {
-			Name:  "BUILDKITE_AGENT_NAME",
-			Value: "buildkite",
-		}, {
-			Name:  "BUILDKITE_CONTAINER_ID",
-			Value: "0",
-		}, {
-			Name:  "BUILDKITE_COMMAND",
-			Value: "cp -r ~/.ssh /workspace/.ssh && chmod -R 777 /workspace",
```

Comment:
Note: I've removed this command in the new definition of the checkout container below.